mitodl/ocw-studio
# Handle level import, add term and year to metadata

#### Pre-Flight checklist


#### What are the relevant tickets?
Related to https://github.com/mitodl/ocw-hugo-projects/issues/89

#### What's this PR do?
Adjusts import to handle updated format for level, and also include data for term and year. The import code should remain backwards compatible for a short while so imports don't break while ocw-to-hugo runs.

#### How should this be manually tested?
 - Run `./manage.py import_ocw_course_sites -b ocw-to-hugo-output-qa --limit 1`. This should take about 5 minutes
 - In the Django shell run `WebsiteContent.objects.get(website__name="1-00-introduction-to-computers-and-engineering-problem-solving-spring-2012", type="sitemetadata").metadata`. You should see a "year" field with `None` in the metadata.

When we switch to the newer ocw-to-hugo output it should also work without changes, but we can't test that yet.
